keps/sig-architecture/1544-kep-artifacts/README.md

```diff
@@ -0,0 +1,176 @@
+
+## Release Signoff Checklist
+
+- [ ] Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] KEP approvers have approved the KEP status as `implementable`
+- [ ] Design details are appropriately documented
+- [ ] Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] Graduation criteria is in place
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://github.com/kubernetes/enhancements
+[kubernetes/kubernetes]: https://github.com/kubernetes/kubernetes
+[kubernetes/website]: https://github.com/kubernetes/website
+
+## Summary
+
+KEP artifacts are difficult to navigate.  It's hard to find which KEPs are in
+which states and to know what is considered undecided.  This proposal aims to
+define several small, incremental steps that can be taken to make these
+problems a bit better.
+
+## Motivation
+
+A group of people from sig-arch started a KEP reading group.  We found it very
+difficult to keep track of what was happening with KEPs.  We identified a
+number of specific suggestions to make consuming KEPs more friendly.  History
+can be found in [this thread](https://groups.google.com/forum/#!msg/kubernetes-sig-architecture/SZ7UcFuOtCI/PQlTeSCICAAJ).
+
+### Goals
+
+To make it easier for humans and automation to find, reference, and consume
+KEPs.
+
+### Non-Goals
+
+This proposal does not cover all of the issues, but does address some specific
+pain points, incrementally.
+
+## Proposal
+
+### 1) Change the names of KEPs
+
+The date-format in the name is not very human friendly and doesn't do anything
+for machines.  Instead, we propose to use the issue number of the enhancement
+issue as the KEP number.  KEP names would now be:
+`1234-short-human-friendly-string`.  This makes them easier to reference and
+remember: "KEP 1234" is easier than "KEP 20190903".  We will not retroactively
+change existing KEPs, so as to not break external links to them.
+
+### 2) Standardize an intra-KEP denotation for "not resolved"
+
+There is a perception that "merge and iterate" can be used to avoid dissent.
+If one topic is under debate, but other parts are acceptable, it should be
+possible to merge a KEP and iterate on the debatable point.  However, without
+reading the entire history of PRs on a KEP, exactly what is "under debate" is
+not clear.
+
+Requirements:
+  - not an HTML comment (so it is visible in the rendered markdown)
+  - unambiguous
+  - can wrap large bodies of text
+
+Parts of a KEP that are being merged without consensus shall be denoted as
+follows:
+
+```
+<<[UNRESOLVED optional text]>>
```

Comment:
Bikeshed nit: mixing 2 different bracket types irks me, and is annoying to type.

Reply:
propose something better

Reply:
ref: https://github.com/kubernetes/enhancements/pull/1545#discussion_r379872258